pkg/webhook/webhook.go

```diff
@@ -60,3 +66,11 @@ func AddToManager(mgr manager.Manager, poolManager *pool_manager.PoolManager) er
 	}
 	return nil
 }
+
+func cipherSuites() []string {
+	cipherSuitesEnv := os.Getenv("TLS_CIPHERS")
```

Comment:
What happens if TLS_CIPHERS happens to be empty?

Reply:
Then the decision is delegated to go tls package.